.travis.yml

```diff
@@ -1,13 +1,22 @@
+dist: trusty
 sudo: required
 
+language: node_js
+node_js:
+    - lts/*
+
 services:
     - docker
 
+addons:
+  apt:
+    packages:
+      - docker-ce
+
 script:
     - if [ "true" = "${SHELLCHECK-}" ]; then shellcheck *.sh ; fi
     - if [ -n "${NODE_VERSION-}" ]; then ./test-build.sh $NODE_VERSION ; fi
     - if [ "true" = "${DOCTOCCHECK-}" ]; then
-         nvm install node &&
          npm i -g doctoc  &&
          cp README.md README.md.tmp &&
          doctoc --title='## Table of Contents' --github README.md &&
```

Comment:
We could `npx doctoc` to avoid the `install` as well?

Reply:
I don't think `npx` is pre-installed. And wouldn't that just install `doctoc` anyway?

Reply:
`npx` is preinstalled as long as npm is 5.4.0 (or something like that) or higher, which it is in node 8 installations.

And it would not install it, it's added to a temp place. The thought was more to save a command than to save an installation anyways

Reply:
Oh, interesting. I didn't realize it was added to newer NPM

Reply:
I think `npx doctoc` is slow, maybe try `yarn` here?

Reply:
yarn has no "install cmd temporarily and run it", see https://github.com/yarnpkg/yarn/issues/3937

Reply:
but seems `npx` be default has no cache for the temporarily install libraries?

Reply:
It uses the npm cache of tarballs, but it doesn't keep the installation around, no. But is that needed here?